extensions/sql-database-projects/src/tools/packageHelper.ts

```diff
@@ -57,48 +56,19 @@ export class PackageHelper {
 	 */
 	public async addPackageToAFProjectContainingFile(filePath: string, packageName: string, packageVersion?: string): Promise<void> {
 		try {
-			const project = await this.getAFProjectContainingFile(filePath);
+			const project = await aFUtils.getAFProjectContainingFile(filePath);
 
 			// if no AF projects were found, an error gets thrown from getAFProjectContainingFile(). This check is temporary until
 			// multiple AF projects in the workspace is handled. That scenario returns undefined and shows an info message telling the
 			// user to make sure their project has the package reference
 			if (project) {
 				await this.addPackage(project, packageName, packageVersion);
+			} else {
+				void vscode.window.showInformationMessage(`To use SQL bindings, ensure your Azure Functions project has a reference to ${constants.sqlExtensionPackageName}`);
```

Comment:
moved this message out from getAFProjectContainingFile() so it wouldn't show when looking for the project for loading the settings